app/ui/legacy/src/main/res/layout/account_list_item.xml

```diff
@@ -1,49 +1,72 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<com.fsck.k9.view.DraggableFrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="?selectableItemBackground"
-    android:clickable="true"
-    android:focusable="true"
-    android:gravity="center_vertical"
-    android:orientation="horizontal"
-    android:paddingLeft="8dp">
-
-    <ImageView
-        android:layout_width="40dp"
-        android:layout_height="40dp"
-        android:contentDescription="@string/account_settings_action"
-        android:padding="8dp"
-        app:srcCompat="?attr/iconSettingsAccount" />
+    android:background="?android:attr/windowBackground"
+    android:foreground="?attr/selectableItemBackground"
+    android:stateListAnimator="@anim/draggable_state_list_anim"
+    tools:ignore="UnusedAttribute">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:paddingLeft="24dp"
-        android:paddingRight="16dp">
+        android:background="?android:attr/windowBackground"
```

Comment:
`DraggableFrameLayout` was added to manage the background color and hold the foreground drawable. It's also the root view of this layout, so it's the one which FastAdapter will listen on for click events. Remove the following attributes from this LinearLayout to make sure clicks get through to `DraggableFrameLayout` and that backgrounds/foregrounds aren't drawn twice:

- android:clickable
- android:focusable
- android:background
- android:foreground